app/_landing_pages/mesh.yaml

```
@@ -44,48 +39,102 @@ rows:
 
   - header:
       type: h2
-      text: "Get Started"
+      text: "Install {{site.mesh_product_name}}"
+    columns:
+      - blocks:
+        - type: structured_text
+          config:
+            blocks:
+              - type: text
+                text: |
+                  You can use the following to get an instance of {{site.mesh_product_name}} in Universal mode running:
+                  
+                  1. Go to the [{{site.mesh_product_name}} packages](https://cloudsmith.io/~kong/repos/{{site.mesh_product_name_path}}-binaries-release/packages/?q=version%3A{{ page.version_data.version }}) page to download and extract the installation archive for your OS, or download and extract the latest release automatically (Linux or macOS):
+                     ```shell
+                     curl -L {{site.links.web}}/mesh/installer.sh | VERSION={{site.data.mesh_latest.version}} sh -
+                     ```
+                  2. To finish installation, add {{site.mesh_product_name}} binaries to path:
+                     ```shell
+                     export PATH=$(pwd)/{{site.mesh_product_name_path}}-{{site.data.mesh_latest.version}}/bin:$PATH
+                     ```
+                     This directory contains binaries for `kuma-dp`, `kuma-cp`, `kumactl`, `envoy` and `coredns`
```

```suggestion
                     This directory contains binaries for `kuma-dp`, `kuma-cp`, `kumactl`, `envoy`, and `coredns`.
                     
                     
                     For other installation options, see below.
```
There's no Mesh install page, so I just put "see below" to direct them to the tiles? - maybe there's a better way to do this? Any ideas?

---

@lena-larionova There's a Mesh install page: https://kongdeveloper.netlify.app/mesh/install/ But it does contain mostly the same content as what I put here. We could remove the install card below (should we remove the separate install page too?) and then we could add a note about seeing below for additional, longer quickstarts for the different deployment options.

---

actually that's a good point - I want a `/mesh/install/` url for optics/easy to remember URL purposes, but right now it's just a duplicate. So maybe we just redirect /mesh/install to the mesh landing page.

Post-GA, we could build something more like what we have for Gateway install: https://kongdeveloper.netlify.app/gateway/install/

---

Made an issue here for the "nicer" install page post GA: https://github.com/Kong/developer.konghq.com/issues/1326